packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -378,7 +376,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   }
 
   void _handleDragUpdate(DragUpdateDetails details) {
-    if (!_isActive || _moveController.isAnimating) {
+    if (!_dragUnderway || _moveController.isAnimating) {
```

Comment:
The `_isActive` getter is only used in the context of `(!_isActive || _moveController.isAnimating)`, which expands to:

```dart
(!(_dragUnderway || _moveController.isAnimating) || _moveController.isAnimating)
```

![truth table](https://github.com/flutter/flutter/assets/10457200/5de3edfe-bbc7-4244-b240-c1332d9746ae)

I made a truth table to try to understand what was going on. Hopefully now it won't be necessary 🙂